src/sky/pro/average/Employee.java

```diff
@@ -0,0 +1,50 @@
+package sky.pro.average;
+
+public class Employee {
+    final private String fullName;
+    private int salary;
+    private int department;
+    private static int counter = 0;
+
+
+    public static int countAdd() {
+        counter++;
+        return counter;
```

Comment:
```suggestion
        return ++counter;
```
в данной форме мы сначала увеличиваем переменную, а затем возвращаем её значение.